_test/cargo_clean_all.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+status=0
+repo="$(cd "$(dirname "$0")/.." && pwd)"
+
+for ex in "$repo"/exercises/*/*/; do
+  name=$(grep '^name =' "$ex/Cargo.toml" | cut -d\" -f2)
+  if [ -z "$name" ]; then
+    echo "don't know name of $ex"
+    status=1
+    continue
+  fi
+  cargo clean --manifest-path "$ex/Cargo.toml" --package "$name"
```

Comment:
given it only cleans the exercise package and not dependencies, I would understand objections to calling this "clean_all", in which case I would have to think of a better name. `cargo_clean_exercises`?